core/templates/domain/collection/collection-node.model.spec.ts

```diff
@@ -19,6 +19,10 @@
 import { CollectionNode } from
   'domain/collection/collection-node.model';
 
+interface SummaryObject {
+  category?: string
```

Comment:
Why is category optional?

Reply:
In the same file, there's a deletion of the category field in summaryObject (https://github.com/oppia/oppia/blob/develop/core/templates/domain/collection/collection-node.model.spec.ts#L61)

Strict checks returned "error TS2790: The operand of a 'delete' operator must be optional"

Reply:
Could you add a comment into the code explaining this?